docs/usage-examples/find.txt

```diff
@@ -0,0 +1,82 @@
+.. _laravel-find-usage:
+
+=======================
+Find Multiple Documents
+=======================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: find many, retrieve, code example
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can retrieve multiple documents from a collection by calling the ``where()`` method
+on an Eloquent model or a query builder.
```

Comment:
`where` doesn't retrieve the documents. It creates a query builder. That the call on `get()` or `cursor()` that runs the query and returns the results.